Makefile

```diff
@@ -125,16 +125,16 @@ dev_setup:
 	test $(MY_USER)
 	test $(shell id -u -r) -eq 0
 	chown -R $(MY_USER): $(CWD)
-	su - $(MY_USER) -c "make -C $(CWD) submodules"
-	su - $(MY_USER) -c "make -C $(CWD) dev_setup_runtime"
-	su - $(MY_USER) -c "make -C $(CWD)/app/tests testing.id_rsa"
-	su - $(MY_USER) -c "make -C $(CWD) front"
-	su - $(MY_USER) -c "make -C $(CWD) pip"
-	su - $(MY_USER) -c "make -C $(CWD) assets"
+	su -m $(MY_USER) -c "make -C $(CWD) submodules"
```

Comment:
I also had some success with GOPATH=$(GOPATH)
`su - $(MY_USER) -c "GOPATH=$(GOPATH) make -C $(CWD) assets"`
i it helps

Reply:
Yep, that way you're passing only one variable. Using '-m' you're passing all of them, that way it's better because all in all you have to run it with sudo -E (which is same (in case of preserving envs) as su -m)